apps/frontend/src/views/Results.vue

```diff
@@ -297,6 +330,58 @@ export default class Results extends mixins(RouteMixin, ServerMixin) {
   /** Determines if we should make the search bar collapse-able */
   show_search_mobile: boolean = false;
 
+  /** If we are currently showing the search help modal */
+  showSearchHelp = false;
+
+  @Watch('searchTerm')
+  onUpdateSearch(_newValue: string) {
+    if (this.typingTimer) {
+      clearTimeout(this.typingTimer);
+    }
+    this.typingTimer = setTimeout(this.onDoneTyping, 1000);
+  }
```

Comment:
What was the reason for adding the delay here instead of just using computed properties? Was it too slow with computed properties?

Reply:
Yeah it was lagging behind when it ran the search every time you pressed a key. Now it just waits until you're finished typing to search.